### PR TITLE
build sleuthkit under windows

### PR DESCRIPTION
**Pull request by Agile86**
It could be useful to use sleuthkit-related tables under Windows too.
Build sleuthkit (tsk) under windows and make available
_device_file_, _device_hash_, _device_partitions_ tables.

### CODE REVIEW

**Comment by Agile86** [comment]
Usage sample:
image
image
image
image

File: libraries/cmake/source/sleuthkit/config/windows/tsk/tsk_config.h [hunk 1]
```diff
@@ -0,0 +1,285 @@
+/* tsk/tsk_config.h.  Generated from tsk_config.h.in by configure.  */
```
Comment (theopolis) [inline libraries/cmake/source/sleuthkit/config/windows/tsk/tsk_config.h:1]: How was this file generated?

Comment (Agile86) [reply libraries/cmake/source/sleuthkit/config/windows/tsk/tsk_config.h:1]: took it from a "linux/tsk" with one change - disabled option HAVE_UNISTD_H
WIth next commit, I did review each option.

**Comment by theopolis** [comment]
Cool, one last change then I think we are good to go! Can you update the integration test's to also build `device_*` on all Platforms? 

https://github.com/osquery/osquery/blob/master/tests/integration/tables/CMakeLists.txt#L83

**Pull request merged**